packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -1,16 +1,37 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+type BreadcrumbSize = 'small' | 'medium' | 'large';
+type BreadcrumbDividerType = 'chevron' | 'slash';
+/**
+ * Data shared between breadcrumb components
+ */
+export interface BreadcrumbContextValue {
+  /**
+   * Controls size of Breadcrumb items and dividers.
+   *
+   * @default 'medium'
+   */
+  size?: BreadcrumbSize;
+
+  dividerType?: BreadcrumbDividerType;
+}
 export type BreadcrumbSlots = {
+  /**
+   * Root element of the component.
+   */
   root: Slot<'nav'>;
+  /**
+   * Ordered list which contains items.
+   */
   list?: Slot<'ol'>;
 };
 
 /**
  * Breadcrumb Props
  */
-export type BreadcrumbProps = ComponentProps<BreadcrumbSlots> & {};
+export type BreadcrumbProps = ComponentProps<BreadcrumbSlots> & Partial<BreadcrumbContextValue> & {};
```

Comment:
```suggestion
export type BreadcrumbProps = ComponentProps<BreadcrumbSlots>;
```

- Let's avoid pairing between props and `BreadcrumbContextValue`
- `{}` is a [bomb in TS](https://www.typescriptlang.org/play?#code/C4TwDgpgBAglC8UDeUBmB7dAuKBnYATgJYB2A5lAL5QBkylAUAwMbon5QCGCyUARpwI4A5AILCqQA)